apps/central-scan/frontend/src/components/delete_batch_modal.tsx

```diff
@@ -29,9 +29,9 @@ export function DeleteBatchModal({
       content={
         <React.Fragment>
           <P>This action cannot be undone.</P>
-          {deleteBatchMutation.error && (
-            <P color="danger">{`${deleteBatchMutation.error}`}</P>
-          )}
+          {deleteBatchMutation.error ? (
+            <P color="danger">{deleteBatchMutation.error as string}</P>
+          ) : null}
```

Comment:
Why this change?

Reply:
The original, with `@types` upgrades, no throws lint error: 
```
src/components/delete_batch_modal.tsx:30:10 - error TS2746: This JSX tag's 'children' prop expects a single child of type 'ReactNode', but multiple children were provided.

30         <React.Fragment>
            ~~~~~~~~~~~~~~

```

That doesn't make sense to me, because I thought the whole point of a fragment was that it could have multiple children. But if you change it to:

```
          {deleteBatchMutation.error ? (
            <P color="danger">{`${deleteBatchMutation.error}`}</P>
          ) : null}
```
Then it passes, and is also valid and effectively the same. The `as string` part can be removed. f0ac108d1.

Reply:
I think the problem here is that `deleteBatchMutation.error` has type `unknown`. A few ideas for working around this...

Add a type assertion for that value:

```ts
{deleteBatchMutation.error as Error | undefined && <P color="danger">{`${deleteBatchMutation.error}`}</P>}
```

Plug in the Error type in the mutation:

```ts
    return useMutation<void, Error, { batchId: string }>(apiClient.deleteBatch);
```
The problem with this is that TS doesn't support partial inference of type parameters, so you have to explicitly list all of them.